index.html

```diff
@@ -270,87 +329,289 @@
+derived from one's personal data) as well as mechanisms to explain the outcomes of automated
+decisions.
+
+## People's Agents {#user-agents}
+
+The <dfn>user agent</dfn> acts as an intermediary between a [=person=] (its [=user=]) and the web.
+[=User agents=] implement, to the extent possible, the [=rules=] that collective governance
+establishes in favour of individuals. They seek to prevent the creation of asymmetries of
+information, and serve their [=user=] by providing them with automation, to rectity
+[=automation asymmetries=]. Where possible, they protect their [=user=] from receiving
+[=inappropriate=] information.
+
+The [=user agent=] is expected to align fully with the [=person=] using it and operate exclusively
+in that [=person=]'s interest. It is <em>not</em> the [=first party=]. The [=user agent=] serves the
+[=person=] as a <dfn>trustworthy agent</dfn>:
 it always puts that [=person=]'s interest first. In some occasions, this can mean protecting
 that [=person=] from themselves by preventing them from carrying out a dangerous decision,
 or by slowing down the person in their decision. For example, the
 [=user agent=] will make it difficult for that [=person=] to connect to a site if it can't verify
 that the site is authentic. It will check that that [=person=] really intends to expose a
 sensitive device to a page. It will prevent that [=person=] from consenting to the permanent
-monitoring of their behaviour. Its <dfn>user agent duties</dfn> include
-[[?TAKING-TRUST-SERIOUSLY]]:
+monitoring of their behaviour. Its <dfn class="export">user agent duties</dfn> include
+([[?Taking-Trust-Seriously]]):
 
 <dl>
   <dt><dfn class="export">Duty of Protection</dfn></dt>
   <dd>
     Protection requires [=user agents=] to actively protect their [=user=]'s data, beyond
     simple security measures. It is insufficient to just encrypt at rest and in transit,
     but the [=user agent=] must also limit retention, help ensure that only strictly
-    necessary data is collected, and require guarantees from any [=party=] that the user agent can reasonably be aware that it is shared to.
+    necessary data is collected, and require guarantees from any [=party=] that the user agent can
+    reasonably be aware that it is shared to.
   </dd>
   <dt><dfn class="export">Duty of Discretion</dfn></dt>
   <dd>
     Discretion requires the [=user agent=] to make best efforts to enforce
-    [=norms=] by placing limits on the flow and
-    [=processing=] of [=personal data=] that it can reasonably detect. Discretion is not confidentiality or secrecy: trust
+    [=rules=] by placing limits on the flow and
+    [=processing=] of [=personal data=] that it can reasonably detect. Discretion is not
+    confidentiality or secrecy: trust
     can be preserved even when the [=user agent=] shares some [=personal data=], so long as
     it is done in an [=appropriately=] discreet manner.
   </dd>
   <dt><dfn class="export">Duty of Honesty</dfn></dt>
```

Comment:
It is wonderful that this draft speaks to duties. Rights and responsibilities must be organized in balance [<a href="https://www.youtube.com/watch?v=O1rm02PMKq8">O'Neil</a>]. This is a critical missing dimension of an otherwise imbalanced, rights-centric discussion.

Reply:
This is a description of the relationship that obtains between a person and their agent. Agents have duties specifically towards their entrustors because they have inordinate amounts of power in the relationship. Put differently, the risk of a malicious agent to the user is staggering; the risk of a malicious user to the agent are negligible. The duties mirror this.

This is not meant as a general discussion of the entire online system, that would indeed need to balance rights and responsibilities.